src/store/store.js

```diff
@@ -94,6 +109,13 @@ export const actions = {
     }).finally( data => {
```

Comment:
Remove space to be consistent with the how like-wise functions are defined.

Reply:
I think this is an older version too